test/functional/ui/incsubstitution_spec.lua

```diff
@@ -0,0 +1,1374 @@
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local clear, feed, insert = helpers.clear, helpers.feed, helpers.insert
+local execute, expect, eval = helpers.execute, helpers.expect, helpers.eval
+local neq, source, eq = helpers.neq, helpers.source, helpers.eq
+local meths, curbufmeths = helpers.meths, helpers.curbufmeths
```

Comment:
By the way, this is supposed to be “one variable per line” (I personally write it as “one variable per line, sorted by variable name length” just because it looks prettier). Convention was introduced some time after many tests were written.